backend/inspirehep/records/marshmallow/literature/ui.py

```diff
@@ -256,6 +256,8 @@ def get_documents_without_fulltext(self, data):
                 del document["attachment"]
             if "text" in document:
                 del document["text"]
+            if "error" in document:
+                del document["error"]
```

Comment:
I would put it in the Base literature marshamallow to remove it from all API responses.

Reply:
so maybe let's use private field there, these fields are hidden in public schema then 
https://github.com/inspirehep/inspirehep/blob/243f4378683d4ba6357b3adec76d911f8f1422b3/backend/inspirehep/records/marshmallow/literature/base.py#L51

Reply:
I think it does something else https://github.com/inspirehep/inspirehep/blob/243f4378683d4ba6357b3adec76d911f8f1422b3/backend/inspirehep/records/marshmallow/fields/non_hidden.py#L11-L15